test/src/Test/Main.purs

```diff
@@ -84,6 +84,10 @@ config =
   , permissions: []
   }
 
+showContentExpired ∷ forall a. Show a => ExpiredContent a -> String
+showContentExpired =
+  \(ExpiredContent {content, expired}) → "content: " <> show content <> " expired: " <> show expired
```

Comment:
as ExpiredContent is newtype now, you can just add `Show` instance for it. and remove ` <#> map showContentExpired` staff

Reply:
Thanks buddy it's been really helpful, I'll make the other amend you recommended with `isHeadersExpired` and make some tests for it

Reply:
👌